openstack-cloud/src/main/java/jenkins/plugins/openstack/compute/JCloudsSlaveTemplate.java

```diff
@@ -428,6 +435,46 @@ public ListBoxModel doFillNetworkIdItems(@RelativePath("..") @QueryParameter Str
             return m;
         }
 
+        public ListBoxModel doFillFloatingIpPoolIdItems(@RelativePath("..") @QueryParameter String identity,
```

Comment:
@olivergondza Should I add that annotation to the doFillFloatingIpPoolIdItems method? I didn't see that used anywhere else in the source and this is my first time delving into Jenkins plugin development.